prow/pod-utilities.md

```diff
@@ -50,8 +48,29 @@ dumped for automatic upload to GCS upon job completion.
 
 ### How to configure
 
+ProwJobs may request Pod Utility decoration by setting `decorate: true` in their config.
+Example ProwJob configuration:
+```yaml
+
+  - name: pull-job
+    agent: kubernetes
+    context: pull-job
+    always_run: true
+    rerun_command: "/test pull-job"
+    trigger: "(?m)^/test (all|pull-job)\\s*"
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/test-images/bug-finder:1.0
+        command:
+        - "/find-bugs"
```

Comment:
Can command ever be a string instead of a list, like in Dockerfile? If so might want to drop a comment about this (looks like you do verbally below)

Reply:
No this field must be a string slice.  It must be specified here because the pod utilities do not respect the container image's entry point specified in the Dockerfile.

Reply:
Added a clarification that the ENTRYPOINT directive is ignored.